pages/Configuring/Keywords.md

```diff
@@ -103,16 +103,49 @@ Inside of it, put your config options. All options from the `input` category
 
 force_no_accel, follow_mouse, float_switch_override_focus
 
-Properties that change names:
+Properties that change names according to the device name:
 
 ```plain
-touchdevice:transform -> transform
-touchdevice:output -> output
+<touchdevice>:transform -> transform
```

Comment:
whats with the `<>`?

I'd just do touchdevice:transform, tablet:transform -> transform

Reply:
the `<>` is to indicate that it's a placeholder for the actual device name. If found that it is sometimes unclear in the wiki where verbatim text is meant and where a placeholder is meant, which is especially confusing for words that could be used as class names. So I believe that we should indicate placeholders as such, if there's a better way *how* I'm all ear ;).